lib/facter/pe_status_check.rb

```diff
@@ -203,6 +203,22 @@
     { S0022: validity }
   end
 
+  chunk(:S0024) do
+    next unless PEStatusCheck.primary? || PEStatusCheck.replica? || PEStatusCheck.postgres?
```

Comment:
We want this on PuppetDB instances, so we don't need external posgresql and do need compilers.